standalone-metastore/metastore-common/src/main/java/org/apache/hadoop/hive/common/repl/ReplConst.java

```diff
@@ -82,4 +84,10 @@ public class ReplConst {
   public static final String BOOTSTRAP_DUMP_STATE_KEY_PREFIX = "bootstrap.dump.state.";
 
   public static final String READ_ONLY_HOOK = "org.apache.hadoop.hive.ql.hooks.EnforceReadOnlyDatabaseHook";
+
+  /**
+   * Type of failover
+   */
+  public static final String PLANNED_FAILOVER = "PLANNED";
```

Comment:
It's usually better to create an enum for types.
https://docs.oracle.com/javase/tutorial/java/javaOO/enum.html